### ISSUE TITLE
Content Validation - On Demand

### ISSUE THREAD

**Issue by mandolyte** [unfoldingWord/tc-create-app#499]
This issue covers implementation of invoking content validation on-demand.

**Comment by elsylambert** [comment]
@mandolyte I fixed the content and then validated again. Is there any indication to the user if there are no content issues?

**Comment by cckozie** [comment]
Working on the project at https://qa.door43.org/translate_test/en_tn/src/branch/tc01-tc-create-1/en_tn_57-TIT.tsv
I opened the project and clicked on the Validate icon
The download file had these two rows:
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/c2d436e4-76b0-41be-b5a5-10693b0eb353)

I fixed the problem in row 3 (Unable to find original language quote in verse text). Correct highlighting then appeared in UGNT

In fixing the problem in row 2 (Unexpected double spaces), I noticed that all three bullet numbers were followed by double spaces when viewing the markdown text. I removed the extra space following the three numbers, then saved the file.

I again ran the validation and this time the original 2 issues I fixed were still included, and 2 new issues were added:
![image.png](https://images.zenhubusercontent.com/56e1a9316caf56c9d38d8df5/d64b1bb8-7676-46f1-be35-14055adfa7a7)

**Comment by cckozie** [comment]
I loaded a Matthew project and clicked the Validate icon. It took 50 seconds to run the validator on Matthew on my fastest Mac. There was no indication that it was actually running during that time, and I was able to perform other operations in the project while it was running.
I then clicked the Validate icon again, and this time the icon and tooltip stayed "highlighted" and I was not able to perform other operations in the project until the validation was completed.
